src/EnergyPlus/PlantHeatExchangerFluidToFluid.cc

```diff
@@ -959,24 +965,84 @@ namespace PlantHeatExchangerFluidToFluid {
         // second deal with Loop Demand Side
         tmpDmdSideDesignVolFlowRate = FluidHX(CompNum).DemandSideLoop.DesignVolumeFlowRate;
         if (FluidHX(CompNum).DemandSideLoop.DesignVolumeFlowRateWasAutoSized) {
-            if (tmpSupSideDesignVolFlowRate > SmallWaterVolFlow) {
-                tmpDmdSideDesignVolFlowRate = tmpSupSideDesignVolFlowRate;
-                if (PlantFirstSizesOkayToFinalize) FluidHX(CompNum).DemandSideLoop.DesignVolumeFlowRate = tmpDmdSideDesignVolFlowRate;
+            if (PltSizNumDmdSide > 0) {
+                if (tmpSupSideDesignVolFlowRate > SmallWaterVolFlow) {
+                    const int supplyLoopType(PlantSizData(PltSizNumSupSide).LoopType);
+                    const int demandLoopType(PlantSizData(PltSizNumDmdSide).LoopType);
+
+                    Real64 supplyLoopInitTemp(0.0);
+                    Real64 demandLoopInitTemp(0.0);
+
+                    if (supplyLoopType == HeatingLoop) {
+                        supplyLoopInitTemp = HWInitConvTemp;
+                    }
+                    else if (supplyLoopType == CondenserLoop || supplyLoopType == CoolingLoop) {
+                        supplyLoopInitTemp = CWInitConvTemp;
+                    }
+                    else {
+                        assert(false);
+                    }
```

Comment:
Correct initialization temp used here, I just didn't notice when this was corrected.